bootstrap.sh

```diff
@@ -15,12 +27,16 @@ export ARCH=amd64
 # You also need to export your C compiler flavor (gcc, clang, gcc-7...)
 export CC=gcc
 # And build:
-./util/build
+$GOBIN/build
 # See \`build -h' for more information on the build tool.
 
 To enable access to files, create a harvey and none user:
 sudo useradd harvey
 sudo useradd none
 
 none is only required for drawterm/cpu access
+
+Also:
+export HARVEY=$(pwd)
+add $GOBIN to your path
```

Comment:
if you add `$GOBIN` to your path, why are you calling `$GOBIN/build` on line 30?

Reply:
because I did not add GOBIN to the path, the script just recommends it?

Reply:
yea it's just a long cat-eof-style output comment - confused me at first as well when not looking closely 👀 

starting here https://github.com/Harvey-OS/harvey/pull/970/files#diff-3af436e6760a0c5f1e3d16a3384853a8L12
which GitHub cuts off (you need to click the expander for the full context;  can be a bit annoying here sometimes, and helpful in other cases)

Reply:
yup. I was reviewing it using the mobile app missed the folded lines. my bad folks. I apologize

Reply:
those folded things can be hell. 

Thanks for the reviews everyone, I did set modules on, what else needs to change?